### PR TITLE
Mention linting

### PR DESCRIPTION
**Pull request by plaindocs**
Minimal viable pre-commit docs.

---
:books: Documentation previews :books:

- User's documentation (`docs`): https://docs--11445.org.readthedocs.build/en/11445/

- Developer's documentation (`dev`): https://dev--11445.org.readthedocs.build/en/11445/

### CODE REVIEW

File: docs/dev/style-guide.rst [hunk 1]
```diff
@@ -49,6 +49,18 @@ Content
...
 
+Linting
+~~~~~~~
+
+ReStructuredText for both the RTD user documentation and this developer documentation is linted with `pre-commit` for broken links,
+whitespace consistency, and other minor issues.
+
+You can fix many linting issues automatically by installing and running `pre-commit` locally. To run it on content you have have already committed, use the ``--from-ref`` and ``--to-ref`` `flags <https://pre-commit.com/#pre-commit-run>`__.
+
+```
```
Comment (ericholscher) [inline docs/dev/style-guide.rst:60]: Believe this is markdown syntax.

File: docs/dev/style-guide.rst [hunk 2]
```diff
@@ -49,6 +49,18 @@ Content
...
+❯ pre-commit run --from-ref <OLDER_COMMIT> --to-ref <RECENT_COMMIT>
+```
+
```
Comment (ericholscher) [inline docs/dev/style-guide.rst:63]: I often use `pre-commit run --all-files` if we want to run it against all the existing files, I've never used this from/to syntax.

File: docs/dev/style-guide.rst (hunk 1 above)
Comment (plaindocs) [reply docs/dev/style-guide.rst:60]: Urgh, will fix

File: docs/dev/style-guide.rst (hunk 2 above)
Comment (plaindocs) [reply docs/dev/style-guide.rst:63]: That gives me a bunch of modified files though, where as this just fixes my PR ;-) 

But up to you folks

**Review by ericholscher — APPROVED** [review]

**Comment by plaindocs** [comment]
That CircleCI error looks unrelated to me, shall I merge?

**Comment by humitos** [comment]
Yes, don't worry about those tests --they are for the new dashboard and not related with your work. Thanks! I'm merging this PR now 👍🏼

**Pull request merged**